.electron-vue/release.js

```diff
@@ -113,8 +113,8 @@ function release () {
         synopsis: pkg.description,
         target: [
           //'deb',
```

Comment:
This setting here might need revert for travis. I changed it to `rpm` only due to my local dev-env. 
And about the copyright, I guess we can set it to be `The Electron-SSR Authors`